spec/models/cart_spec.rb

```diff
@@ -32,4 +32,10 @@
 
     expect(cart.formatted_total_price).to eq("36.00")
   end
+
+  it "can return total number of items in the cart" do
+    cart = Cart.new( {"1" => 5, "2" => 9} )
```

Comment:
This is annoying, but just to get rid of the hound, you should do `Cart.new("1" => 5, "2" => 9)`